assets/js/aplus.js

```diff
@@ -606,3 +606,31 @@ $.fn.highlightCode = function(options) {
         });
     };
 })(jQuery, window, document);
+
+
+/**
+ * prevent double submit of forms
+ */
+
+$(function() {
+
+    $.fn.preventDoubleSubmission = function() {
+  $(this).on('submit',function(e){
+    var $form = $(this);
+
+    if ($form.data('submitted') === true) {
+      // Previously submitted - don't submit again
+      e.preventDefault();
+    } else {
+      // Mark it so that the next submit can be ignored
+      $form.data('submitted', true);
+    }
+  });
+
+  // Keep chainability
+  return this;
+};
+
+$('form').preventDoubleSubmission();
```

Comment:
This could hook to `#exercise-page-content form`. On chapter page, this would be problematic, but there we open modal dialog when submit is clicked. So I think it's good enough to do this kind of limiting only on old style exercise pages. (this the id in jquery selector).